File: Projects/10/Java/birds.java
/* Birds.java illustrates inheritance and polymorphism.
 *
 * Completed by: Emily Costa
 * Date: 4/27/2020
 *********************************************************/

public class Birds {
  
    public static void main(String[] args) {
        System.out.println("\nWelcome to the Bird Park!\n");

        Duck bird1 = new Duck("Donald");
        bird1.print();
  
        Goose bird2 = new Goose("Mother Goose");
        bird2.print();
  
        Owl bird3 = new Owl("Woodsey");
        bird3.print();

        Penguin bird4 = new Penguin("Penny");
        bird4.print();

        Ostrich bird5 = new Ostrich("Orville");
        bird5.print();

        Kiwi bird6 = new Kiwi("Kevin");
        bird6.print();

        System.out.println();
        System.out.println("Hope you had fun! Come back soon!");
        System.out.println();
      }
}

File: Projects/10/Ruby/birds.rb
# birds.rb | Tests out the menagerie of bird classes.
#
# Begun by: Dr. Adams, for CS 214 at Calvin College.
# Completed by: Emily Costa
# Date: 4/27/2020
######################################################

require './Bird.rb'
require './Walkingbird.rb'
require './Flyingbird.rb'
require './Duck.rb'
require './Goose.rb'
require './Owl.rb'
require './Penguin.rb'
require './Ostrich.rb'
require './Kiwi.rb'

puts "\nWelcome to the Bird Park!\n\n"

bird1 = Duck.new "Donald"
bird1.print

bird2 = Goose.new "Mother"
bird2.print

bird3 = Owl.new "Woodsey"
bird3.print

bird4 = Penguin.new "Penny"
bird4.print

bird5 = Ostrich.new "Orville"
bird5.print

bird6 = Kiwi.new "Kevin"
bird6.print

puts "\nHope you enjoyed your time!\n"

puts "\n\n"